src/Translator/doc/index.rst

```diff
@@ -71,6 +71,10 @@ For a better developer experience, TypeScript types definitions are also generat
 Then, you will be able to import those JavaScript translations in your assets.
 Don't worry about your final bundle size, only the translations you use will be included in your final bundle, thanks to the `tree shaking <https://webpack.js.org/guides/tree-shaking/>`_.
 
+.. note::
+
+    This package requires the `translator` to be enabled in your Symfony application. If you don't use the `translator` service, the warmup command will not generate any translations.
```

Comment:
> This package requires the `translator` to be enabled

Is `translator` a package, a service of something else? In the next sentence it is written that it's a service, I think it should be explicit in the first sentence too.

Reply:
Code should have double quotes, other wise it is displayed like this:

> ![image](https://github.com/user-attachments/assets/1d6d0a60-e47c-48a1-a93b-807f81311204)

Reply:
Side note: This PR may be reverted soon:

- #2060

Reply:
I've updated this note in #2061